packages/app/src/app/pages/Dashboard/Content/routes/MyContributions/EmptyContributions.tsx

```diff
@@ -0,0 +1,75 @@
+import track from '@codesandbox/common/lib/utils/analytics';
+import {
+  ArticleCard,
+  CreateCard,
+  Element,
+  Stack,
+} from '@codesandbox/components';
+import { EmptyPage } from 'app/pages/Dashboard/Components/EmptyPage';
+import React from 'react';
+
+const DESCRIPTION =
+  "Open any open-source repository on CodeSandbox and just click 'Branch' to start contributing.<br />Don't worry about forking and setting up a new repository. We'll take care of everything for you.";
```

Comment:
I would drop the `just` from the copy: https://dev.to/fordevs-community/contributing-better-docs-with-carolyn-stransky-a0o

Reply:
@alexnm that makes sense! thank you for pointing it out